src/utils/zipcodeConversion.js

```diff
@@ -0,0 +1,15 @@
+import zipcodes from 'zipcodes';
+
+const zipcodeConversion = zipcode => {
+  try {
+    let convertedZipcode = zipcodes.lookup(zipcode);
+    let error =
+      'Sorry, zipcode is invalid. Please check your input and try again.';
+    if (!convertedZipcode) throw error;
+    return convertedZipcode;
+  } catch (error) {
+    console.log(error);
```

Comment:
Why are we just logging the error?
The error should probably be returned so it can be handled by the code using this utility function.